scripts/src/report/templates/components/results/1-candidate-species.html

```diff
@@ -22,6 +22,7 @@ <h2>Identification of candidate species</h2>
   </em>
 </p>
 
+{% if not bold %}
 <div class="row my-5">
```

Comment:
This looks like it will cause broken HTML tags - it will remove `<div class="row">` but not the closing `</div>` tag.
Probably works ok just to move `{% if not bold %}` down a line so the `row` element is always there. Bootstrap `.col` elements should always be inside a `.row` element.